### ISSUE TITLE
Turn sample app into full-screen or "kiosk" browser

### ISSUE THREAD

**Issue by ocram** [delight-im/Android-AdvancedWebView#88]
A common use case for web views on Android is building a full-screen browser, which often displays a single site only (without address bar), and which may even be locked down into "kiosk" mode [[1]](https://en.wikipedia.org/wiki/Kiosk_software) [[2]](https://play.google.com/store/apps/details?id=com.procoit.kioskbrowser).

We arguably have the best implementation of a web view's core features here already. However, this library is only for developers, and completely unsuited for end-users, since it requires integration into a separate application that is then built from source.

It would be easy -- and make a lot of sense -- to turn the sample application, which currently resides in `Source/sample`, into a generic full-screen or "kiosk" browser.

Right now, the sample application is not that helpful. It just serves the purpose of showing a basic integration of this library into an app. This way, developers who want to use this library can look up how to structure or implement certain parts of the app.

Turning the app into a proper single-site browser would give a real use case to the sample app, which would become a convenient website wrapper that can be used for free by everybody. We could even provide a pre-compiled APK here.

Help and feedback on the required features would be much appreciated. Thanks!

**Comment by digitalprecision** [comment]
Yes please! As a newb android developer I found the sample code to be somewhat difficult to implement into a "kiosk" browser. I think just the basics would be fine. A title bar which shows the page title fed from <title>, and a circular progress bar should be enough. A nice to have would be a chrome type of menu where the user clicks the upper right hand and a few options pop up to refresh, share, etc.
